### PR TITLE
Added Byens Server Media

### PR DESCRIPTION
**Pull request by Simonmaribo**
## Type of change

- [x] Add new directory for a server
- [x] Update directory of a server
- [x] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by Simonmaribo** [comment]
image

I hope the player count is high enough for it to be added.

File: minecraft_servers/byen/manifest.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "nice_name": "Byen",
+    "direct_ip": "mc.byen.net",
+    "server_wildcards": [
+      "byen.net",
+      "%.byen.net"
+    ],
+    "social": {
+      "web": "https://byen.net",
+      "web_shop": "https://butik.byen.net",
+      "web_support": "https://discord.byen.net",
```
Comment (JakobBruening) [inline minecraft_servers/byen/manifest.json:12]: Discord is not supported. Only faq/web support pages are accepted for "web_support".

File: minecraft_servers/byen/manifest.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      "discord": "https://discord.byen.net"
```
Comment (JakobBruening) [inline minecraft_servers/byen/manifest.json:13]: As custom discord invite links are only supported for LabyMod partners at the moment.
Please use a discord invite link.

Documentation reference: https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url

**Review by JakobBruening — CHANGES_REQUESTED** [review]

**Comment by JakobBruening** [comment]
Closed until issues are resolved.

**Comment by Simonmaribo** [comment]
Alright - a new PR has been created.

**Comment by Simonmaribo** [comment]
https://github.com/LabyMod/server-media/pull/446

**Pull request closed without merging**